module4/projects/solo_stretch_tech/index.md

```diff
@@ -3,14 +3,137 @@ title: Solo Stretch Tech
 layout: page
 ---
 
-## Learning Goals
+## Overview + Learning Goals
 
+In this project, students will create a roadmap for and begin a 10-week-long project. That’s right - 10 weeks! 
 
-## Overview
+Turing grads often struggle post-graduation with establishing solid routines, keeping their technical skills sharp, and upskilling in new technical areas. The goals of this project are meant to directly address these areas:
+- Students will solidify a plan for the first 8 weeks of post-graduate life.
+- Students will create a project that they are enthusiastic about, giving them a strong story for interviews.
+- Students will continue to solidify programming concepts and add more skills to their resume.
 
+For the most part, you will spend the next 2 weeks planning and getting your project set up so that you have something tangible to work on after graduation. You will only contribute a little bit of code to this project before graduation.  
 
-## Expectations
+## MVP (Minimum Viable Product)
 
+You will be designing this application from the ground up. For the most part, you have control over what you build, but we want to help set you up for success by defining a few requirements for the product. We don't want you choosing a project that is too small to last over the next 10 weeks and we don't want you starting to code when you haven't done your due diligence planning. For your MVP, we expect:
+- A clearly defined app idea with a specific audience
+- A large enough scope to take 10+ weeks
+- Two stretch technologies (1 BE and 1 FE) implemented
+- The frontend and backend should "talk" at least 3 different places in the app, with at least one request mutating the data (POST, PATCH, PUT, DELETE)
 
-## Evaluation Details
+## Stretch Technologies
 
+**We expect you to implement 1 stretch tech from each menu below (2 total).** If you would like to add more, you can, but be careful not to bite off too much. If there is a technology you're interested in using that is not on these lists, let your instructor know and we'll do our best to make it work.
+
```

Comment:
Do we want to add Auth? CI/CD? Where would those fit into the "menus" below?

Reply:
I was actually thinking CI/CD should be required, not stretch, and due with the day 7 repo deliverables. Auth could go either way. I think I'd say OAuth is stretch, regular auth feels iffier to me.